draft-ietf-privacypass-protocol.md

```diff
@@ -273,31 +282,31 @@ round sees the server generate a commitment. The second round sees the
 server issue a token to the client.
 
 ~~~
-  Client(pkS, m, info)                        Server(skS, pkS)
-  ------------------------------------------------------------
+  Client(pkS, m, info, cMetadata)                        Server(skS, pkS, sMetadata)
```

Comment:
Let's spell out these parameters? That is, `cMetadata` => `clientMetadata`.

Reply:
Also, shouldn't both parties have client and server metadata as input?

Reply:
Yes! changed everywhere.